spec/classes/corosync_spec.rb

```diff
@@ -54,6 +54,32 @@
       end
     end
 
+
+    [ :package_corosync, :package_pacemaker, :version_corosync, :version_pacemaker ].each { |package_param|
+      context "new-style package parameter $#{package_param} mixed with deprecated $packages parameter" do
```

Comment:
Good thinking!

Can we also get tests for the new parameters behaving as they should when (not) specified?